priv/gettext/errors.pot

```diff
@@ -93,3 +93,48 @@ msgstr ""
 
 msgid "must be equal to %{number}"
 msgstr ""
+
+#, elixir-autogen, elixir-format
+#: lib/t/accounts/profile.ex:279
+msgid "%{label} can't be blank"
```

Comment:
once ios starts using [error response](https://github.com/getsince/test5/blob/a6dbeeba17126034d52e6377b929fdd88bf0877a/Since/StoryEdit/StoryEditCoordinator.swift#L144-L150), these would need to be translated